src/components/results/shortcircuit/shortcircuit-analysis-result.type.ts

```diff
@@ -40,11 +38,46 @@ export interface SCAResultFault {
     feederResults: SCAResultFaultFeederResult[];
 }
 
-export interface ShortcircuitAnalysisResult {
-    resultUuid: UUID;
-    writeTimeStamp: Date;
-    faults: SCAResultFault[];
-}
+type Pageable = {
+    sort: {
+        sorted: boolean;
+        empty: boolean;
+        unsorted: boolean;
```

Comment:
why having `sorted` and `unsorted` at same time? they're always of opposed value no?

Reply:
I think that we will not use it so I delete it, thank you